src/utils/github_updater.rs

```diff
@@ -121,12 +122,11 @@ fn get_github_fields(path: &str) -> Result<GitHubFields> {
             .unwrap_or(0),
         forks: obj.get("forks_count").and_then(|d| d.as_i64()).unwrap_or(0),
         issues: obj.get("open_issues").and_then(|d| d.as_i64()).unwrap_or(0),
-        last_commit: time::strptime(
+        last_commit: DateTime::parse_from_rfc3339(
             obj.get("pushed_at").and_then(|d| d.as_str()).unwrap_or(""),
-            "%Y-%m-%dT%H:%M:%S",
         )
-        .unwrap_or_else(|_| time::now())
-        .to_timespec(),
+        .map(|datetime| datetime.with_timezone(&Utc))
+        .unwrap_or_else(|_| Utc::now()),
```

Comment:
Can you summarize why we're adding a timezone here? It looks like this changed from the previous behavior.

Reply:
`time::now()` gets the current timestamp which is then turned into a zoneless time, we're doing the same thing but keeping the zone